packages/app/src/contracts.ts

```diff
@@ -1,4 +1,5 @@
-import ExampleNFTGoerli from "@web3-scaffold/contracts/deploys/goerli/ExampleNFT.json";
```

Comment:
I'm trying to figure out a good way of allowing the chain to be specified here. 🤔